src/poliastro/plotting/tisserand.py

```diff
@@ -0,0 +1,142 @@
+            Body to be plotted Tisserand
+        vinf_array: ~astropy.units.Quantity
+            Desired Vinf for the flyby
+
+        Returns
+        -------
+        r_a: ~np.ndarray
+            Array of arrays for all possible apoapsis radius
+        r_p: ~np.ndarray
+            Array of arrays for all possible periapsis radius
+
+        Note
+        ----
+        The algorithm for generating Tisserand plots is the one depicted in
+        "Preliminary Trajectory Design of a Mission to Enceladus" by David
+        Falcato Fialho Palma, section 3.6
+
+        """
+
+        # Generate mean orbital elements Earth
+        body_rv = get_mean_elements(body).to_vectors()
+        R_circ, V_circ = norm(body_rv.r), norm(body_rv.v)
+
+        # Generate non-dimensional velocity and alpha span
+        vinf_array /= V_circ
+        alpha_array = np.linspace(0, np.pi)
+
+        # Construc the mesh for any configuration
+        v_inf, alpha = np.meshgrid(vinf_array, alpha_array)
+
+        # Solving for non-dimensioanl orbital elements
+        a_sc = 1 / (1 - v_inf ** 2 - 2 * v_inf * np.cos(alpha))
+        e_sc = np.sqrt(1 - 1 / a_sc * ((3 - 1 / a_sc - v_inf ** 2) / (2)) ** 2)
+
+        # Convert to dimensional orbital elements
+        r_a = a_sc * (1 + e_sc) * R_circ
+        r_p = a_sc * (1 - e_sc) * R_circ
+
+        # TODO: what about Tisserans with:
+        #   - Energy vs r_a
+        #   - Orbital period vs r_a
+
+        return r_a, r_p
+
+    def plot(self, body, vinf_span, num_contours=10, color=None):
+        """ Plots body Tisserand for given amount of solutions within Vinf span
+
+        Parameters
+        ----------
+        body: ~poliastro.bodies.Body
+            Body to be plotted Tisserand
+        vinf_span: tuple
+            Minimum and maximum Vinf velocities
+        num_contours: int
+            Number of points to iterate over previously defined velocities
+
+        """
+
+        # Generate a lisnpace for all possible Vinf speeds
+        vinf_array = np.linspace(vinf_span[0], vinf_span[-1], num_contours)
+
+        # Obtain Tisserand lines
+        r_a, r_p = self.solve_tisserand(body, vinf_array)
+
+        # Get actual body color
+        if not color:
+            color = BODY_COLORS[body.name]
+
+        # Add lines to the custom axis
+        self.ax.plot(r_a, r_p, color)
+
+        # TODO: contour lines would help to plot inline contour levels
+
+
+if __name__ == "__main__":
+
+    # All Solar System Bodies
+    from poliastro.bodies import Mercury, Venus, Earth, Mars, Jupiter, Saturn
+
+    SS_BODIES = [
+        Mercury,
+        Venus,
+        Earth,
+        Mars,
+    ]
+
+    # Generate Tisserand
+    tp = TisserandPlotter()
+
+    for ss in SS_BODIES:
+
+        # This example works fine
+        tp.plot(ss, (1, 10) * u.km / u.s, num_contours=10)
+
+        # TODO: BUG APPEARS IF THE UPPER BOUND FOR VINF > 10 KM/S
+        #tp.plot(ss, (1, 11) * u.km / u.s, num_contours=11)
```

Comment:
I can reproduce the issue. It looks like there's an x value going to 0, or similar.